rtl/__init__.py

```diff
@@ -436,6 +435,18 @@ def interactive_controlfile(self):
     def interactive_controlfile(self,value): 
         self._interactive_controlfile = value
 
+    @property
+    def vlogsimargs(self):
```

Comment:
Added simulation arguments for verilog simulation (vsim command, or vvp with Icarus). There already existed a property called `vlogcompargs` so this property follows its naming scheme